core/CMakeLists.txt

```diff
@@ -49,7 +46,7 @@ if(NOT DEFINED CORE_LIB_NAME)
     set(CORE_LIB_NAME core)
 endif()
 
-add_library(${CORE_LIB_NAME} ${CORE_LIB_TYPE} ${FOUND_SOURCES} ${FOUND_HEADERS})
```

Comment:
Yeah I think we can revert that part, except if there was other reasons to remove it?